cfme/configure/configuration/region_settings.py

```diff
@@ -136,8 +135,8 @@ class CompanyCategoriesAllView(RegionView):
     @property
     def is_displayed(self):
         return (
-            self.company_categories.is_active() and
-            self.table.is_displayed
```

Comment:
Why removing this table check?

Reply:
@mshriver It was an option change means removal of a `table is_displayed` accepted, but when I was checking `CompanyCategoriesAllView` I found the table is not a unique thing which we always want while writing `is_displayed` method so, I just replace unique thing means checking `company_categories` tag is active or not with table. I am assuring you it will effect on any test.